pkgs/os-specific/linux/nvidia-drivers/default.nix

```diff
@@ -0,0 +1,229 @@
+{ stdenv, fetchurl
+
+, buildConfig ? "all"
+, channel ? null
+
+# Kernelspace dependencies
+, kernel ? null
+, nukeReferences
+
+# Userspace dependencies
+, xlibs
+, zlib
+
+, libsOnly ? false
+
+, nvidiasettingsSupport ? true
+  , atk ? null
+  , gdk_pixbuf ? null
+  , glib ? null
+  , pango ? null
+  , gtk2Support ? false
+    , gtk2 ? null
+  , gtk3Support ? true
```

Comment:
I remember Eelco didn't like the dependency on gtk3 by default (a few months ago). I think we should start preferring gtk3 at *some* point &ndash; even in general though I don't think we need a mass switch in cases where both versions are well supported.